terrawrap/utils/graph.py

```diff
@@ -0,0 +1,98 @@
+""" Holds graph utilities"""
+import os
+import networkx
+
+from typing import List, Tuple, Any
+
+
+def has_cycle(graph: networkx.DiGraph) -> bool:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Either all return statements in a function should return an expression, or none of them should.](https://app.codacy.com/manual/amplify-education/terrawrap/pullRequest?prid=4739316)